iroh-net/src/magicsock.rs

```diff
@@ -900,7 +927,7 @@ impl Inner {
         Poll::Ready(Ok(()))
     }
 
-    fn try_send_derp(&self, url: &DerpUrl, node: PublicKey, contents: DerpContents) -> bool {
+    fn try_send_derp(&self, url: &DerpUrl, node: PublicKey, contents: DerpContents) -> Poll<bool> {
```

Comment:
Should rename this to `poll_send_derp` now that it returns `Poll`? And maybe even make it take the `Context` as argument, register the waker in there and return `Poll<Result<()>>` instead of `Poll<bool>`, would align it more to other `poll_send` methods.

Reply:
we call it from places where there is no context available, which is why I am not passing it as argument

Reply:
renamed